decidim-core/lib/decidim/settings_manifest.rb

```diff
@@ -96,6 +96,7 @@ class Attribute
       }.freeze
 
       attribute :type, Symbol, default: :boolean
+      attribute :preview
```

Comment:
can you add a comment explaining the attribute please?
```suggestion
      # Expects a proc that will have to return fake data to be used as a preview for the current Attribute
      attribute :preview
```